File: Golang/jarak/jarak.go
package main

import "fmt"

func main() {
	var jarak, kecepatan, waktu float32

	fmt.Println("Program Menghitung Jarak Berdasarkan Kecepatan dan Waktu")
	fmt.Print("Masukkan kecepatan: ")
	fmt.Scan(&kecepatan)
	fmt.Print("Masukkan waktu: ")
	fmt.Scan(&waktu)

	jarak = kecepatan * waktu
	fmt.Println("Jika kecepatan", kecepatan, "km/jam dan waktu tempuh", waktu, "jam, maka jarak yang ditempuh adalah", jarak, "km.")
	// fmt.Printf("Jika kecepatan %v km/jam dan waktu tempuh %v jam, maka jarak yang ditempuh adalah %v km.\n", kecepatan, waktu, jarak)
	// fmt.Print("Jika kecepatan ", kecepatan, " km/jam dan waktu tempuh ", waktu, " jam, maka jarak yang ditempuh adalah ", jarak, " km.\n")
}

File: Python/jarak.py
print("Program Menghitung Jarak Berdasarkan Kecepatan dan Waktu Tempuh")
kecepatan = float(input("Masukkan kecepatan: "))
waktu = float(input("Masukkan waktu: "))

jarak = kecepatan * waktu
print(
    "Jika kecepatan "
    + str(kecepatan)
    + " km/jam dan waktu tempuh "
    + str(waktu)
    + " jam, maka jarak yang ditempuh adalah "
    + str(jarak)
    + " km."
)
# output = "Jika kecepatan {} km/jam dan waktu tempuh {} jam, maka jarak yang ditempuh adalah {} km."
# print(output.format(kecepatan, waktu, jarak))
